test/Table_Tests/src/IO/Excel_Spec.enso

```diff
@@ -1005,45 +988,47 @@ spec =
+        group_builder.specify "Patchy table" <|
             check_table (file.read (Excel (Cell_Range "Sheet1!D1"))) ["A", "B", "F"] [[1,2,4], [4,4,Nothing], [6,Nothing,6]]
             check_table (file.read (Excel (Cell_Range "Sheet1!D2"))) ["D", "E", "F"] [[1,2,4], [4,4,Nothing], [6,Nothing,6]]
             check_table (file.read (Excel (Cell_Range "Sheet1!E"))) ["B"] [[4,4,Nothing,Nothing,Nothing,Nothing]]
             check_table (file.read (Excel (Cell_Range "Sheet1!E1"))) ["B", "F"] [[4,4,Nothing], [6,Nothing,6]]
             check_table (file.read (Excel (Cell_Range "Sheet1!E2"))) ["E", "F"] [[4,4,Nothing], [6,Nothing,6]]
 
-        Test.specify "Single cell" <|
+        group_builder.specify "Single cell" <|
             check_table (file.read (Excel (Cell_Range "Sheet1!H1"))) ["H"] [["Single Cell"]]
             check_table (file.read (Excel (Cell_Range "Sheet1!H2"))) ["H"] [[]]
 
-        Test.specify "Single line" <|
+        group_builder.specify "Single line" <|
             check_table (file.read (Excel (Cell_Range "Sheet1!J1"))) ["J", "K", "L"] [["Just"],["Some"],["Headers"]]
 
-        Test.specify "Growing table" <|
+        group_builder.specify "Growing table" <|
             check_table (file.read (Excel (Cell_Range "Sheet1!N1"))) ["A", "Full", "Table", "Q"] [["Hello","World",Nothing,"Extend"],[1,Nothing,"Gap",3],[2,2,"Here",5],[Nothing,Nothing,"To","Hello"]]
             check_table (file.read (Excel (Cell_Range "Sheet1!O1"))) ["Full", "Table", "Q"] [[1,Nothing,"Gap",3],[2,2,"Here",5],[Nothing,Nothing,"To","Hello"]]
             check_table (file.read (Excel (Cell_Range "Sheet1!O2"))) ["O", "P", "Q"] [[1,Nothing,"Gap",3],[2,2,"Here",5],[Nothing,Nothing,"To","Hello"]]
 
-        Test.specify "Should handle blank headers without warnings" <|
+        group_builder.specify "Should handle blank headers without warnings" <|
             check_table (file.read (Excel (Cell_Range "Sheet1!D1"))) ["A", "B", "F"] [[1,2,4], [4,4,Nothing], [6,Nothing,6]]
 
-        Test.specify "Should handle duplicate headers with warnings" <|
+        group_builder.specify "Should handle duplicate headers with warnings" <|
             action = file.read (Excel (Cell_Range "Sheet1!S1")) on_problems=_
             tester = check_table _ ["DD", "DD 1"] [[1,3], [2,4]]
             problems = [Duplicate_Output_Column_Names.Error ["DD"]]
             Problems.test_problem_handling action problems tester
 
-    # Cleanup any leftovers from previous runs
-    enso_project.data/"transient" . list "out*" . each .delete
-    spec_write "xlsx" 'TestSheet.xlsx'
-    spec_write "xls" 'TestSheetOld.xls'
+    spec_write suite_builder "xlsx" 'TestSheet.xlsx'
+    spec_write suite_builder "xls" 'TestSheetOld.xls'
+
+main =
+    suite = Test.build suite_builder->
+        add_specs suite_builder
+    suite.run_with_filter spec_filter="should write a table to non-existent file as a new sheet with headers; and return the file object on success"
```

Comment:
I think this filter should not be left over in the repo, right?

@Akirathan can you please remove it in a next PR?